modules/develop/pages/manage-topics/cloud-topics.adoc

```diff
@@ -0,0 +1,89 @@
+= Cloud Topics
+:description: Cloud Topics are Kafka topics that enable users to trade off latency for lower costs.
+:page-beta: true
+// tag::single-source[]
+
+ifndef::env-cloud[]
+[NOTE]
+====
+include::shared:partial$enterprise-license.adoc[]
+====
+endif::[]
+
+Starting in v25.3, Redpanda provides Cloud Topics to support multi-modal streaming workloads in the most cost-effective way possible. While standard Kafka xref:config-topics.adoc[topics] are ideal for latency-sensitive, high-throughput workloads (for example, for audit logs), Cloud Topics are optimized for high-throughput, cost-sensitive workloads that can tolerate higher latencies. Instead of replicating every byte across expensive network links, Cloud Topics leverage durable, inexpensive Cloud object storage (S3, GCS, MinIO) as the primary mechanism for backing up data. The difference in response time is typically 500ms, which is only impactful for latency-sensitive applications.
+
+== Prerequisites
+
+- xref:get-started:rpk-install.adoc[] v25.3 or later.
+ifndef::env-cloud[]
+- xref:manage:tiered-storage.adoc#set-up-tiered-storage[Set up] and xref:manage:tiered-storage.adoc#enable-tiered-storage[enable Tiered Storage] on your Redpanda cluster.
++
+[NOTE]
+====
+Be sure to set the following cluster-level properties to `false` to disable Tiered Storage for new topics:
+
+* config_ref:cloud_storage_enable_remote_write,false,properties/object-storage-properties[]
+* config_ref:cloud_storage_enable_remote_read,false,properties/object-storage-properties[]
+
+For details, see xref:manage:tiered-storage.adoc#enable-tiered-storage-for-a-cluster[Enable Tiered Storage for a cluster].
+====
+
+- xref:manage:tiered-storage.adoc#configure-object-storage[Configure object storage].
+- You must have an Enterprise license.
+
+To check if you already have a license key applied to your cluster:
+
+[,bash]
+----
+rpk cluster license info
+----
+endif::[]
+
+== Limitations
+
+Not all Redpanda features are available when using Cloud Topics (beta). The following features are not supported in this beta release:
+
+- Kafka compaction
+- Iceberg topics
+- Topic recovery
+- Remote Read Replicas
+
+Also note that Cloud Topics do not work in simple Docker containers unless Cloud storage is set up in them.
+
+IMPORTANT: You should expect to throw away any cluster running Cloud Topics (beta) after it becomes generally available (GA). Redpanda will be unable to continue using data that was generated by the beta version of this feature. Also, any topic specified to be a Cloud Topic cannot subsequently be converted back to a standard Kafka topic.
```

Comment:
'may be unable'

"IMPORTANT: You should expect to throw away any cluster running Cloud Topics (beta) after it becomes generally available (GA). Redpanda may be unable to continue using topics creates with the beta version of this feature and you may be unable to upgrade to future versions.  Also, any topic specified to be a Cloud Topic cannot subsequently be converted back to a standard Redpanda topic using local or tiered storage"